Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM emberjs/website-dev:latest
+
+# Customizations for this project here.
+# e.g.: if you have dependencies in addition to what the base image provides:
```

Comment:
I am not convinced if the above comment provides much value, maybe it would be better to state here why `Gemfile*` are copied into a container?